### PR TITLE
Strings

### PR DESCRIPTION
**Pull request by yulkadev**

### CODE REVIEW

File: Exercises/strings/strings.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+
+//Assign string with unicode to a variable.
+
+const varWithUnicode = '\u00A9';
+console.log(varWithUnicode);
+
+//Declare a variable with string with escaped symbols.
+
+let varWithEscapedSymbols = 'It\'s not a problem.';
```
Comment (lizarusi) [inline Exercises/strings/strings.js]: `const` preferable

File: Exercises/strings/strings.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+console.log(varWithEscapedSymbols);
+
+//Declare a variable which will contain name of your favourite animal.
+
+let myFavoriteAnimalName = 'Tom';
```
Comment (lizarusi) [inline Exercises/strings/strings.js]: const preferable, or you are going to change your taste :D

File: Exercises/strings/strings.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+//Log a message My favourite animal is with dynamic variable from previous task.
+
+console.log(`My favourite naimal name is ${myFavoriteAnimalName}`);
+
+//Declare a variable which contain name and surname of your favourite writer in lowercase. Log his/her name with capitals first letters.
+
+const myFavouriteWriter = 'conan doyle';
+
+function capitalize(author) {  
+  return author.split(' ').map(el => el.charAt(0).toUpperCase() + el.substring(1)).join(' ');
```
Comment (lizarusi) [inline Exercises/strings/strings.js]: the `+` I recommend to replace on backticks

Comment (lizarusi) [inline Exercises/strings/strings.js]: this is a good implementation! But I want you to add one more example without map, assuming we have only one space between name and surname

Comment (lizarusi) [reply Exercises/strings/strings.js]: el.charAt(0) -> el[0]

File: Exercises/strings/strings.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+//should contain at leas one special character.
+//Make function which will generate password suggestion based on passed password, and required strongest.
+
+function passwordStrong(passwordString) {
+
+  if(!passwordString.match(/[A-Z]/)) {
+    console.log('Password should contain at least one capital letter');
+  }
+
+  if(passwordString.length === 0 || passwordString.length <= 8) {
```
Comment (lizarusi) [inline Exercises/strings/strings.js]: do you actually need to check `passwordString.length === 0` ? :)

File: Exercises/strings/strings.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+//Additional
+
+//Declare a function which will receive password string. Check is password strong enough:
+//should contain at least one capital letter;
+//length no less than 8 characters.
+//should contain at leas one special character.
+//Make function which will generate password suggestion based on passed password, and required strongest.
+
+function passwordStrong(passwordString) {
+
```
Comment (lizarusi) [inline Exercises/strings/strings.js]: odd space

File: Exercises/strings/strings.js (hunk 1 above)
Comment (yulkadev) [reply Exercises/strings/strings.js]: changed to const

File: Exercises/strings/strings.js (hunk 2 above)
Comment (yulkadev) [reply Exercises/strings/strings.js]: :) changed to const

File: Exercises/strings/strings.js (hunk 5 above)
Comment (yulkadev) [reply Exercises/strings/strings.js]: removed empty row

File: Exercises/strings/strings.js (hunk 3 above)
Comment (yulkadev) [reply Exercises/strings/strings.js]: el.charAt(0) -> el[0] - changed

Comment (yulkadev) [reply Exercises/strings/strings.js]: added another implementation

File: Exercises/strings/strings.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+console.log(capitalize(myFavouriteWriter));
+
+// second implementation of capitalise words function without map()
+
+function capitalize2(str)
+{
+  console.log(str.replace(/\w\S*/g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();}));
+}
+
+capitalize_Words(myFavouriteWriter);
```
Comment (Mike-Savin) [inline Exercises/strings/strings.js]: wrong function called

File: Exercises/strings/strings.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+console.log(`My favourite naimal name is ${myFavoriteAnimalName}`);
+
+//Declare a variable which contain name and surname of your favourite writer in lowercase. Log his/her name with capitals first letters.
+
+const myFavouriteWriter = 'conan doyle';
+
+function capitalize(author) {  
+  return author.split(' ').map(el => el[0].charAt(0).toUpperCase() + el.substring(1)).join(' ');  
+}
```
Comment (Mike-Savin) [inline Exercises/strings/strings.js]: 👍

File: Exercises/strings/strings.js [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+}  
+
+console.log(capitalize(myFavouriteWriter));
+
+// second implementation of capitalise words function without map()
+
+function capitalize2(str)
+{
+  console.log(str.replace(/\w\S*/g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();}));
+}
```
Comment (Mike-Savin) [inline Exercises/strings/strings.js]: Good example but seems like it is just copied without even formatting.
Anyway, Liza meant implementation using `slice` method. Please write implementation using slice that supports only two words of unknown length

**Pull request merged**